blog.html

```diff
@@ -0,0 +1,114 @@
+  <body>
+    <header>
+      <img src="images/adoptadog-logo.svg" class="logo" alt="Dog sitting next to AdoptaDog text">
+      <p>Seattle's premiere dog adoption agency</p>
+      <img src="images/cart-icon.svg" class="cart" alt="shopping cart">
+      <button class=checkout-button><a href="checkout.html">checkout</a></button>
+    </header>
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="dogs.html">Dogs</a></li>
+        <li><a href="blog.html">Blog</a></li>
+      </ul>
+    </nav>
+    <section>
+      <h1>Adopt a Blog</h1>
+      <h2>Traveling with your dog</h2>
+      <img src="images/blog-1.jpg" alt="">
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
+        incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+        exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+        dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+        Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit
+        anim id est laborum.
+      </p>
+
+      <h2>How to walk multiple dogs</h2>
+      <img src="images/blog-2.jpg" alt="">
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
+        incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+        exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+        dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+        Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit
+        anim id est laborum.
+      </p>
+
+      <h2>Teach your dog to fetch!</h2>
+      <img src="images/blog-3.jpg" alt="">
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
+        incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+        exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+        dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+        Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit
+        anim id est laborum.
+      </p>
+    </section>
+    <footer>
+      <img src="images/adoptadog-logo.svg" alt="Dog sitting next to AdoptaDog text">
+      <p>
+        dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+        Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit
+        anim id est laborum.
+      </p>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+      <p>@2018 AdoptaDog LLC | Seattle, WA</p>
```

Comment:
Another way to handle the copyright element is using the `&copy'` element. It can be a little more predicatble

`<p>&amp;2018 AdoptaDog LLC | Seattle, WA</p>`